versions/3.0.md

```diff
@@ -1128,20 +1128,20 @@ A request body with a referenced model definition.
       "schema": {
         "$ref": "#/components/schemas/User"
       },
-      "examples": [ "http://foo.bar/examples/user-example.json" ]
+      "examples": [ "http://foo.bar/other-api.yaml#/components/examples/user-example.json" ]
```

Comment:
ok, good feedback - I've updated the PR, and also added a missing JSON block, and also named the `/components/example/` items more uniformly. 

I also fixed some cases where the spec
had `/components/schemas/pet` in some places, `/components/schemas/Pet` in others.